docs/Bread.md

```diff
@@ -0,0 +1,89 @@
+## What is BREAD?
+
+BREAD is the community currency for the Breadchain ecosystem which exists on Gnosis Chain. All BREAD is created through the [Bread Crowdstaking Application](https://www.notion.so/Crowdstaking-Application-9f233bc2fb1e419ebeb58a2809b21658?pvs=21) which anyone with xDAI on Gnosis Chain is able to use to have some BREAD for themselves.
+
+```mermaid
```

Comment:
I know we love diagrams, but is this really providing value? I have two problems with it: (1) It's essentially recreating the structure of the contract is a second place, which means it now needs to be maintained in two places if any changes are made. If we change the contract and forget to change this (which in my experience happens A LOT which stuff like this) it actually detracts from peoples understanding of the system. (2) It's not obvious what all the random symbols in this diagram mean, so it makes for a diagram that is sort of impossible to fully understand. Especially without a "key" for the symbols.